content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/aging.rst

```diff
@@ -0,0 +1,100 @@
+======================
+Inventory aging report
+======================
+
+.. |SVLs| replace:: :abbr:`SVLs (stock valuation layers)`
+
+The inventory aging report evaluates all items in stock, providing insights into potentially sunken
+purchase costs and delays in profitability.
+
+Create customized pivot tables to analyze product, operation types, month, or company breakdowns.
+This helps identify products in stock that are at risk of passing their expiration or viability
+dates, or instances of rot/decay for fast-expiring items.
+
+.. note::
+   The :guilabel:`Reporting` menu in *Inventory* is only accessible to users with :doc:`admin access
```

Comment:
*Nitpick*– this shouldnt have a guilabel yet since it is not the active view :)
```suggestion
   The *Reporting* menu in *Inventory* is only accessible to users with :doc:`admin access
```